dll/win32/imageres/imageres_version.rc

```diff
@@ -0,0 +1,13 @@
+/*
+ * PROJECT:     imageres
+ * LICENSE:     LGPL-2.1+ (https://spdx.org/licenses/LGPL-2.1+)
+ * PURPOSE:     Image Resource
+ * COPYRIGHT:   Copyright 2020 Katayama Hirofumi MZ (katayama.hirofumi.mz@gmail.com)
+ */
+
+#include "imageres_version.h"
+
+#define WINE_FILEVERSION WINE_FILEVERSION_MAJOR,WINE_FILEVERSION_MINOR,WINE_FILEVERSION_BUILD,WINE_FILEVERSION_PLATFORMID
+#define WINE_FILENAME_STR "imageres.dll"
```

Comment:
Why? Do you plan to send this dll to wine or to use their imageres?

Reply:
I'm Wine-friendly. If they want this file, I will provide it.

Reply:
I'm Wine-friendly. If they want this file, I will provide it.